pkg/procmatch/default_catalog.go

```diff
@@ -293,6 +293,14 @@ var DefaultCatalog IntegrationCatalog = IntegrationCatalog{
 			"redis-server",
 		},
 	},
+	IntegrationEntry{
```

Comment:
Odd - I thought this was generated using the integrations-core manifests (https://github.com/DataDog/integrations-core/blob/master/elastic/manifest.json#L19-L21)

Is that not the case?

Reply:
Oh it does look like it's generated per the go-generate comment at the top of this file. Is that what you did?